### DISCUSSION TITLE
Cannot archive repository

### DISCUSSION THREAD

**Discussion by iqfareez** [community/community#18949] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/60868965/174524115-4bf55161-e111-4c41-9dbd-f5b936d42769.png)

When I click the Archive button, no response, the window stay opens like that.

The error from the DevTools console:
![image](https://user-images.githubusercontent.com/60868965/174524154-8dd376b7-238c-489c-9752-ea876bfe8b63.png)

```
Blocked form submission to '/iqfareez/bmi_calculator-Flutter/settings/archive' because the form's frame is sandboxed and the 'allow-forms' permission is not set.
```

**Comment by aenq** [comment]
Add the 'allow-forms' .
"One important thing to note is that the sandboxing flags applied to a frame also apply to any windows or frames created in the sandbox. This means that we have to add allow-forms to the frame's sandbox, even though the form only exists in the window that the frame pops up." - https://web.dev/sandboxed-iframes/

**Comment by Hugoonreplit** [comment]
Also happens to me. Check the Github docs it always works for me.

https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories

**Comment by iqfareez** [comment, marked as answer]
This problem just fixed by itself. No modification needed.
